lib/iris/coords.py

```diff
@@ -1677,13 +1677,15 @@ def _new_points_requirements(self, points):
         if points.size > 1 and not iris.util.monotonic(points, strict=True):
             raise ValueError('The points array must be strictly monotonic.')
 
+    def _points_getter(self):
+        return super(DimCoord, self)._points_getter().view()
```

Comment:
Why aren't we doing this for AuxCoords too?

Reply:
Makes sense. I'll change it.